src/Build/Evaluation/IntrinsicFunctions.cs

```diff
@@ -480,6 +497,26 @@ internal static bool VersionLessThanOrEquals(string a, string b)
             return SimpleVersion.Parse(a) <= SimpleVersion.Parse(b);
         }
 
+        private static object ParseNuGetFramework(string tfm)
+        {
+            return NuGetFramework.GetMethod("Parse", new Type[] { typeof(string) }).Invoke(null, new object[] { tfm });
```

Comment:
Can the `GetMethod`-like calls be cached? Feels like they should be but I don't have perf numbers to back it up so it can definitely wait.